pos_main.py

```diff
@@ -0,0 +1,162 @@
+"""This script facilitates plus one scaling for disperse volume"""
+
+from scale.gf_plus_one_scale import *
```

Comment:
import * is not good practice, it will import everything from that module including variables. Import only required names